src/components/AddTransaction/index.js

```diff
@@ -91,26 +81,24 @@ export default function AddTransaction() {
 				></input>
 
 				<label htmlFor="categorieMenu">Select a fitting category</label>
-				<input
+
+				<select
```

Comment:
Yeah, we love `<select>` 🙂
In this case  it's a better choice instead of using `<datalist>`

Reply:
yeah, makes more sense for the user. Thank you for the hint 👏🏻